[2023-07-15 11:04:16,971] numexpr.utils - INFO - NumExpr defaulting to 8 threads.
[2023-07-15 11:04:20,700] root - INFO - Training pipleine config: TrainingPipelineConfig(artifact_dir='C:\\Users\\Sumeet Maheshwari\\Desktop\\end to end project\\sales_store\\Store_sales_forcasting_Timeseries\\Sales_Forecasting_Artifact\\Artifact')
[2023-07-15 11:04:20,700] root - INFO - Data Ingestion Config : DataIngestionConfig(dataset_download_url='https://github.com/sumeet0701/data-dump/blob/main/outlier_clean.zip?raw=true', tgz_download_dir='C:\\Users\\Sumeet Maheshwari\\Desktop\\end to end project\\sales_store\\Store_sales_forcasting_Timeseries\\Sales_Forecasting_Artifact\\Artifact\\data_ingestion\\2023-07-15-11-04-16\\tgz_data', raw_data_dir='C:\\Users\\Sumeet Maheshwari\\Desktop\\end to end project\\sales_store\\Store_sales_forcasting_Timeseries\\Sales_Forecasting_Artifact\\Artifact\\data_ingestion\\2023-07-15-11-04-16\\raw_data', ingested_data_dir='C:\\Users\\Sumeet Maheshwari\\Desktop\\end to end project\\sales_store\\Store_sales_forcasting_Timeseries\\Sales_Forecasting_Artifact\\Artifact\\data_ingestion\\2023-07-15-11-04-16\\ingested_data') 
[2023-07-15 11:04:20,701] root - INFO - 
******************** Data Ingestion log started ********************

[2023-07-15 11:04:20,701] root - ERROR - 
                        error occured in script: 
                        [C:\Users\Sumeet Maheshwari\Desktop\end to end project\sales_store\Store_sales_forcasting_Timeseries\store_sales\pipeline\training_pipeline.py] at 
                        try block line number: [89] execption block line number: [99]
                        error message: [
                        error occured in script: 
                        [C:\Users\Sumeet Maheshwari\Desktop\end to end project\sales_store\Store_sales_forcasting_Timeseries\store_sales\pipeline\training_pipeline.py] at 
                        try block line number: [38] execption block line number: [41]
                        error message: [
                        error occured in script: 
                        [C:\Users\Sumeet Maheshwari\Desktop\end to end project\sales_store\Store_sales_forcasting_Timeseries\store_sales\components\data_ingestion.py] at 
                        try block line number: [23] execption block line number: [26]
                        error message: [
                        error occured in script: 
                        [C:\Users\Sumeet Maheshwari\Desktop\end to end project\sales_store\Store_sales_forcasting_Timeseries\store_sales\components\db_operation.py] at 
                        try block line number: [26] execption block line number: [52]
                        error message: [[Errno 2] No such file or directory: 'C:\\Users\\Sumeet Maheshwari\\Desktop\\end to end project\\sales_store\\Store_sales_forcasting_Timeseries\\env.yaml']]]]
[2023-07-15 11:04:20,705] root - INFO - 
******************** Data Ingestion log completed ********************

